src/applications/vaos/appointment-list/redux/reducer.js

```diff
@@ -53,6 +56,18 @@ const initialState = {
   expressCareFacilities: null,
 };
 
+function getTestFacilityId(facilityId) {
```

Comment:
@narin I know you did something like this on a recent PR. Do you think the transformer is the right place for this if we need to reference it from here as well?

Reply:
What do you think of having it as an export in the Facility service (`facility/index.js?)`

Reply:
Assuming you mean the Location service, yeah, I think that makes sense.

Reply:
Yup, sorry meant Location.